internal/tjson/schema_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tjson
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/FerretDB/FerretDB/internal/util/testutil"
+)
+
+func TestSchemaMarshalUnmarshal(t *testing.T) {
+	expected := Schema{
+		Title:       "users",
+		Description: "FerretDB users collection",
+		Properties: map[string]*Schema{
```

Comment:
Is one test case enough for everything? I see many types are not supported yet. Is the idea to add more fields to this test case when these types become supported?

Reply:
Yes. See #683